rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,134 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# TL Quarterly Planning Process
+
+## Summary
+
+This document details the a quarterly process by which TLs will contribute to the overall DCP quarterly roadmapping
+process via a Technical Objectives RFC, presented during the TL+PM Planning Call.
+
+## Author(s)
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+## Shepherd
```

Comment:
If you are the shepherd, you can duplicate your name here or delete this section. If you need a shepherd, I would be happy to volunteer.

Reply:
I shall do my best to shepherd myself 😆

Reply:
Done :)